### PR TITLE
index: add upcoming and blog sections

### LINKED ISSUE #484: Integrating blog.openshift.com Wordpress content from into Commons Site
All the content on the blog.openshift.com site that is OpenShift Commons related has now been tagged with a new category "OpenShift Commons".

We would like to add a section to the home page of the commons.openshift.org site that pulls the latest blogs with that tag in LIFO order with a link to the rest at the bottom of the list in a style similar to either of these below screenshots. 
screen shot 2017-07-28 at 3 03 51 pm
screen shot 2017-07-28 at 3 03 18 pm

### LINKED ISSUE #485: Feature the upcoming events more prominently on the home page of commons.openshift.org
add a new section called "UpComing Events" above the "Find Your Commonality" section to the home page with the agenda style calendar 

below is the widget to embed:

<script class="ai1ec-widget-placeholder" data-widget="ai1ec_superwidget" data-action="agenda">
  (function(){var d=document,s=d.createElement('script'),
  i='ai1ec-script';if(d.getElementById(i))return;s.async=1;
  s.id=i;s.src='//live-timely-objzoywz.time.ly/?ai1ec_js_widget';
  d.getElementsByTagName('head')[0].appendChild(s);})();
</script>

### LINKED ISSUE #496: Change the title above the briefings calendar on the home page
titlechangefromevents2briefings

Change the title from "UpComing Events" to "Upcoming Briefings" 
Change the "More Events ->" button to "More Briefings ->"

(first stage of changes to the events section which can be done in this iteration)

### PR DESCRIPTION
**Pull request by jiri-fiala**
* adds Upcoming Events section, closes #485
* adds Blog section, closes #484
* renames Events to Briefings, closes #496

Requested-by: Diane Mueller-Klingspor
Signed-off-by: Jiri Fiala <jfiala@redhat.com>

### CODE REVIEW

File: source/quotes.html.erb [hunk 1]
```diff
@@ -22,7 +22,7 @@ description: Supporting quotes from OpenShift Commons participants.
           </li>
           <li>/</li>
           <li>
-            <a href="/quotes.html">Participant Quotes</a>
+            <a href="/quotes.html">Testimonials</a>
```
Comment (wgordon17) [inline source/quotes.html.erb]: If we're renaming to testimonials, can we change `quotes.html` to `testimonials.html` as well? I prefer the naming to match. We can then include an .htaccess redirect.

File: source/layouts/_navbar.erb [hunk 2]
```diff
@@ -9,6 +9,9 @@
           <li>
             <a href="/index.html#interests">Interests</a>
           </li>
+          <li>
+            <a href="/index.html#blog">Blog</a>
```
Comment (wgordon17) [inline source/layouts/_navbar.erb]: I can't believe I've never noticed this before, but can we drop the `/index.html` from all the nav elements? It makes for a very jarring effect when clicking on the items. The anchor tag by itself provides a much smoother transition.

File: source/index.html.erb [hunk 3]
```diff
@@ -35,178 +35,139 @@ description:  Where users, partners, customers, and contributors come together t
...
+      <h1>
+        Commons
+        <span>Blog Articles</span>
+      </h1>
+    </div>
+  </div>
+  <div class="container wow fadeInUp padding_bottom">
+    <div class="row-fluid">
+      <div class="col-md-8 col-xs-12">
+          <p class="lead">OpenShift Blog is full of great content. If we're to select our top three favorite blog posts, we'd recommend you to have a look on these...</p>
```
Comment (wgordon17) [inline source/index.html.erb]: I think this reads better as 

> The OpenShift Blog is full of great content. Take a look at our top three recommended blog posts.

File: source/css/_custom.scss [hunk 4]
```diff
@@ -794,5 +794,65 @@ ul.participants.list {
...
+  padding-top: 20px;
+
+  &> a {
+	  font-size: 18px;
+  }
+}
+
+// blog section
+#blog {
+	background-color: rgb(245, 245, 245);
```
Comment (wgordon17) [inline source/css/_custom.scss]: Please use the `info_resalt` class to apply the gray background as used on other sections.

File: source/quotes.html.erb (hunk 1 above)
Comment (jiri-fiala) [reply source/quotes.html.erb]: done

File: source/layouts/_navbar.erb (hunk 2 above)
Comment (jiri-fiala) [reply source/layouts/_navbar.erb]: The navbar partial is used on other pages than index as well, so it needs some more work than just dropping `index.html`. I didn't notice jarring when using the links, but there's apparent reflow effect caused by the calendar widget, when scrolling or navigating using links before it's loaded. I added `min-height` to that row to reduce it.

File: source/index.html.erb (hunk 3 above)
Comment (jiri-fiala) [reply source/index.html.erb]: done

File: source/css/_custom.scss (hunk 4 above)
Comment (jiri-fiala) [reply source/css/_custom.scss]: Done. The white-gray-white section ordering is "broken" by the first two sections - they need white background, the form and calendar look bad on gray. Then, blog section follows, which looks ok with the gray background and I kinda liked it to be merged with the testimonials in gray, so I did broke it intentionally, to make the participants stand out a bit, but the ordering could definitely be arranged alternatively. Please ping me to discuss the layout.

File: source/quotes.html.erb (hunk 1 above)
Comment (wgordon17) [reply source/quotes.html.erb]: I appreciate you updating the filename....even though we have .htaccess in place for the redirect, would it also make sense to rename the breadcrumb here?

**Pull request merged**